staubli_rx160_support/tests/roslaunch_test.xml

```diff
@@ -6,4 +7,12 @@
   <group ns="test_rx160__">
     <include file="$(find staubli_rx160_support)/launch/test_rx160.launch"/>
   </group>
+
+  <group ns="load_rx160l__">
```

Comment:
+1
Could you point me to the link (if there is one) that describes this convention?